var/spack/repos/builtin/packages/symengine/package.py

```diff
@@ -35,7 +36,9 @@ class Symengine(Package):
     version('0.1.0', '41ad7daed61fc5a77c285eb6c7303425')
     version('develop', git='https://github.com/symengine/symengine.git')
 
-    variant('flint',        default=True,
+    variant('boost',        default=True,
```

Comment:
Thanks for the tip! I'll revert to `gmp` by default then and make the other compatibility changes that you suggest. Renaming the `boost` option is also probably a good idea to make it transparent what its used for.